### PR TITLE
network: fix a typo which causes the "Primary" select menu to not show up

### PR DESCRIPTION
**Pull request by subhoghoshX**

### CODE REVIEW

File: pkg/networkmanager/team.jsx [hunk 1]
```diff
@@ -170,7 +170,7 @@ export const TeamDialog = ({ connection, dev, settings }) => {
                         {team_balancer_choices.map(choice => <FormSelectOption value={choice.choice} label={choice.title} key={choice.choice} />)}
                     </FormSelect>
                 </FormGroup>}
-                {runner == "active-backup" && <FormGroup fieldId={idPrefix + "-primary-select"} label={_("Primary")}>
+                {runner == "activebackup" && <FormGroup fieldId={idPrefix + "-primary-select"} label={_("Primary")}>
```
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:173]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19003-20230623-080134-a47b976e-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
Nice! the [failing pixel test](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19003-20230623-080134-a47b976e-fedora-38-networking/pixeldiff.html#TestTeam-testBasic-networking-team-settings-dialog) even proves that it works. I'll force-push your branch to update it (as that requires some privileges)

**Comment by martinpitt** [comment]
> as that requires some privileges

... which you should have since last week apparently, I just got told. Anyway, you can do it next time :grin:

File: pkg/networkmanager/team.jsx (hunk 1 above)
Comment (cockpituous) [inline pkg/networkmanager/team.jsx:173]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19003-20230623-085735-84dc87dd-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)

**Review by martinpitt — APPROVED** [review]

**Pull request merged**